src/applications/simple-forms/40-0247/pages/veteranPersonalInfo.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {
+  dateOfBirthSchema,
+  dateOfDeathSchema,
+  currentOrPastDateUI,
+  fullNameNoSuffixSchema,
+  fullNameNoSuffixUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    'ui:title': (
+      <h3 className="vads-u-margin-y--0">
+        Tell us who you’re requesting a Presidential Memorial Certificate for
+      </h3>
+    ),
+    veteranFullName: fullNameNoSuffixUI(),
+    veteranDateOfBirth: currentOrPastDateUI('Date of birth'),
```

Comment:
we also have dateOfBirth and dateOfDeath patterns explicitly.  In those, I think the error message is different.

Reply:
Refactored to use dateOfBirth & dateOfDeath patterns.